src/btree/bt_sync.c

```diff
@@ -341,6 +341,12 @@ __sync_page_skip(
 
     *skipp = false; /* Default to reading */
 
+    /* Skip all deleted pages as they are no longer required for the checkpoint. */
```

Comment:
Could you expand the comment? Something like: "Skip deleted pages as they are no longer required for the checkpoint. The checkpoint never needs to review the content of those pages - if they should be included in the checkpoint the existing page on disk contains the right information and will be linked into the checkpoint as the internal tree structure is built."

Reply:
Thanks. Updated the comment as per the suggestion.